config/scout.php

```diff
@@ -72,6 +72,21 @@
 
     'soft_delete' => false,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Auto Identify User
+    |--------------------------------------------------------------------------
+    |
+    | This option allows you to control whether to automatically
+    | identify the currently authenticated user with a search
+    | search engine of choice. It is turned off by default.
+    |
+    | Supported engines: "algolia"
+    |
+    */
+
+    'user' => env('SCOUT_USER', false),
```

Comment:
I would probably change this for:

- `'user_analytics' => env('SCOUT_USER_ANALYTICS', false),`
- or `'identify_user' => env('SCOUT_IDENTIFY_USER', false),`

I like the first better I think.

Reply:
I went with the latter 😁